src/components/PlanCard/index.tsx

```diff
@@ -0,0 +1,84 @@
+import { Button } from 'components/Button'
+import {
+  ButtonContainer,
+  Container,
+  MinorPlanType,
+  OfferStyle,
+  PlanTypeStyle,
+  PriceContainer,
+  PriceStyle,
+  SignStyle,
+  TextContainer,
+} from './styles'
+
+export interface PlanCardProps {
+  planType: 'Semanal' | 'Mensal' | 'Anual'
+  price: string
+  haveOffer?: boolean
+  offer: string
+  image: string
+  buttonText: string
+}
+
+export const PlanCard = ({
+  planType,
+  price,
+  haveOffer,
+  offer,
+  image,
+  buttonText,
+}: PlanCardProps) => {
+  return (
+    <div
+      className={Container}
+      style={{
+        backgroundImage: `url(${image})`,
+        backgroundRepeat: 'no-repeat',
+        opacity: 0.9,
+        objectFit: 'contain',
+      }}
+      data-testid="planCard"
+    >
+      {haveOffer && (
+        <div data-testid="offer" className={OfferStyle}>
+          {offer}%
```

Comment:
Coloca dentro de um span
```suggestion
          <span>{offer}%</span>
```